File: platforms/python/examples/scene/material/alpha_mode.py
from fragmentcolor import AlphaMode, Material

foliage = Material.pbr().alpha_mode(AlphaMode.Mask).alpha_cutoff(0.3)

glass = Material.pbr().base_color([0.9, 0.95, 1.0, 0.25]).alpha_mode(AlphaMode.Blend)

solid = Material.pbr().alpha_mode(AlphaMode.Opaque)

File: platforms/kotlin/examples/scene/material/alpha_mode.kt
import org.fragmentcolor.*

val foliage = Material.pbr().alphaMode(AlphaMode.MASK).alphaCutoff(0.3f)

val glass = Material.pbr().baseColor(listOf(0.9f, 0.95f, 1.0f, 0.25f)).alphaMode(AlphaMode.BLEND)

val solid = Material.pbr().alphaMode(AlphaMode.OPAQUE)